nixos/modules/services/networking/iwd.nix

```
@@ -36,7 +36,7 @@ in
 
       description = ''
         Options passed to iwd.
-        See [here](https://iwd.wiki.kernel.org/networkconfigurationsettings) for supported options.
+        See `man iwd.config` for supported options.
```

```suggestion
        See {manpage}`iwd.config(5)` for supported options.
```

We have support for manpages in the docs. :slightly_smiling_face:

---

nice, thanks !